src/main/resources/assets/minestuck/blockstates/pink_stone_column.json

```diff
@@ -0,0 +1,10 @@
+{
+  "variants": {
+    "facing=up":    { "model": "minestuck:block/pink_stone_column" },
+    "facing=down":  { "model": "minestuck:block/pink_stone_column"},
+    "facing=east":  { "model": "minestuck:block/pink_stone_column", "y": 90, "x": 90 },
+    "facing=south": { "model": "minestuck:block/pink_stone_column", "y": 180, "x": 90 },
+    "facing=west":  { "model": "minestuck:block/pink_stone_column", "y": 270, "x": 90 },
+    "facing=north": { "model": "minestuck:block/pink_stone_column", "y": 0, "x": 90 }
```

Comment:
Is there a reason the chiseled sandstone columns have a unique model orientation when facing down but the rest do not? I imagine this puts these blockstates closer in property to logs and other axis facing blocks that are not truly directional

Reply:
> You may want to make these an instance of MSDirectionalBlock in case people want to have the chiseled face show up either up or down

Was that not what was meant by your comment in the discord?

Reply:
Ah I think I misunderstood how the textures were being used at that point. Then it all comes down to whether you think it makes sense for the players to be able to flip the texture upside down or not with the non chiseled columns you are adding